assets/product.js

```diff
@@ -305,24 +310,29 @@ function showStickyCheckout() {
   stickyCheckout.style.transform = 'translateY(0)';
   // stickyCheckout.style.opacity = '1';
 
-  const makeModalCenter = () => {
-    stickyCheckout.style.left = `${(window.innerWidth - stickyCheckout.offsetWidth) / 2}px`;
-  }
+  // const makeModalCenter = () => {
+  //   stickyCheckout.style.left = `${(window.innerWidth - stickyCheckout.offsetWidth) / 2}px`;
+  // }
 
-  makeModalCenter();
-  window.addEventListener('resize', makeModalCenter);
+  // makeModalCenter();
+  // window.addEventListener('resize', makeModalCenter);
 }
 
 function triggerCheckout(parentId) {
+  $("body").style.overflow = "hidden";
```

Comment:
to  keep the code consistency please make sure to either use `$` handler for all `document.querySelector` selectors